From b0462d0b2fd730c46f1a30f99932ff944e9d23b4 Mon Sep 17 00:00:00 2001
From: James Lu <GLolol@overdrivenetworks.com>
Date: Mon, 28 Nov 2016 07:58:28 -0800
Subject: password: Force sudo usage on Debian

(LP: #1642331)
---
 base/password.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/base/password.py b/base/password.py
index 443403f..12addc4 100644
--- a/base/password.py
+++ b/base/password.py
@@ -33,7 +33,7 @@ from . import pexpect
 PASSWORD_RETRY_COUNT = 3
 
 AUTH_TYPES ={'mepis':'su',
-             'debian':'su',
+             'debian':'sudo',
              'suse':'su',
              'mandriva':'su',
              'fedora':'su',
